story/utils.py

```diff
@@ -5,10 +5,10 @@
 import yaml
 from profanityfilter import ProfanityFilter
 
-YAML_FILE = "../story/story_data.yaml"
+YAML_FILE = "story/story_data.yaml"
 
 
-with open("../story/censored_words.txt", "r") as f:
+with open("story/censored_words.txt", "r") as f:
```

Comment:
We can't do it this way - now if you'll try to run `build_training_data.py`, you'll have FileNotGound (because the path is relative)